Makefile

```diff
@@ -4,7 +4,11 @@ MAKEFLAGS += --jobs=2
 .PHONY: build
 build:
 	opam exec -- dune build --profile=release
+ifdef CI
```

Comment:
`CI` is exported by default in GHA runner: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/variables#default-environment-variables